core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/workspace/runners/deletion/WorkspaceDeletionRunner.scala

```diff
@@ -101,7 +103,9 @@ class WorkspaceDeletionRunner(val samDAO: SamDAO,
           s"Workspace deletion failed [workspaceId=${workspaceId}, jobControlId=${job.jobControlId}, jobType=${job.jobType}]",
           t
         )
-        workspaceRepository.updateState(job.workspaceId.get, WorkspaceState.DeleteFailed).map(_ => Complete)
+        workspaceRepository
+          .setFailedState(job.workspaceId.get, WorkspaceState.DeleteFailed, t.getMessage)
```

Comment:
Shouldn't we append the jobControlID?

Reply:
We discussed this during standup, but responding here too in case we ever need to reference it in the future:
There's no need to append the `jobControlId` here since all of the exceptions that `runStep` will throw include the `jobControlId` in their message -- see lines 129, 141 and 156.